### ISSUE TITLE
VoiceOver on Mac OS is not reading the start time on radio schedules within the title, but it is reading it out when interacting with the duration time element.

### ISSUE THREAD

**Issue by tochwill** [BBC-archive/psammead#3306]
**Describe the bug**
When using the VoiceOver on home page, the StartTimeWrapper is not read out - the time is read out when interacting with the ButtonWrapper at the bottom of the Card (as _well_ as the duration).

This was seen when testing voiceover with Chrome - we should carry this out again on Safari to see if the issues are also present there.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.com/hausa?renderer_env=live
2. Using VoiceOver, navigate to the radio schedules section.

**Expected behaviour**
The start time should be read out as part of the programme title, and the duration should be read on its own within the duration element - as defined here: https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5dc0724850731722a4aa767a

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by benjifellows** [comment]
<img width="607" alt="Screenshot 2020-03-25 at 16 12 26" src="https://user-images.githubusercontent.com/54984834/77558953-7e933780-6eb3-11ea-8bc9-2ec844698497.png">

When running VoiceOver on Safari, I am unable to replicate this behaviour. As per the A11y testing matrix, the testing should be run in Safari.
https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os
